### PR TITLE
Add R.first function

### PR DESCRIPTION
**Pull request by myrubapa**
Add R.first function

### CODE REVIEW

**Comment by mattgrande** [comment]
I think this is a duplicate of `R.head`

http://ramdajs.com/docs/#head

File: test/last.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ var eq = require('./shared/eq');
 
 describe('last', function() {
 
-  it('returns the first element of an ordered collection', function() {
+  it('returns the last element of an ordered collection', function() {
```
Comment (CrossEye) [inline test/last.js]: Oops!  This should get fixed, regardless of the current PR.

**Comment by CrossEye** [comment]
Thanks for the suggestion.

As @mattgrande said, this is a duplicate of the functionality of `head`.  (At some point, when we'd had only `head` and `tail`, we had requests for one of the the others and we added both `init` and `last` for symmetry.)  So there's no reason for `first`, and we have a no-aliases policy.

Nonetheless, thank you for your interest and for a nice straightforward PR.

I would love a tiny PR focused on fixing the typo in `test/last.js` though, if you're interested.

**Comment by tanrax** [comment]
Wouldn't it be interesting to rename _head_ a _first_?

**Comment by CrossEye** [comment]
@tanrax: No, not to me.  Why do you think it would be an improvement?  `head`/`tail` is a very old convention.

**Pull request closed without merging**